oak-store-spi/src/main/java/org/apache/jackrabbit/oak/json/ParseUtils.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.jackrabbit.oak.json;
```

Comment:
this package does not seems fit for this util imo.

in theory, it doesn't really have anything to do with json parsing but rather with string to Long conversion. 

Maybe someplace in oak-core might be more suitable ?

Reply:
I moved the method to `oak-commons`, in an already existing class `LongUtils`. Initially I was trying to keep the method internal to minimize impact on the public API of Oak, but I think it is better to make it available to other modules in Oak, as this may be useful in other cases.